content/en/threat-model.md

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Threat Model'
+description: ''
+category: 'About sigstore'
+position: 3
+---
+
+We provide a threat model that details the impact of a compromise in any piece of the Sigstore ecosystem. Based on this assessment, we discuss existing mitigations and provide recommendations about the secure configuration of Sigstore. We find that the TUF root of trust provides strong revocation abilities for all Sigstore services in the event of a compromise, but OIDC accounts and issuers used with Sigstore do not have the same revocation abilities. Further, identity and consistency monitoring are critical to many of the security claims of Sigstore, but they are not yet widely applied in practice. We recommend an increased focus on monitoring and 2fa for all OIDC accounts used with Sigstore.
```

Comment:
Nice work!

I may be pedantic here, so don't spin on it, but might be worth defining who 'we' is, as it reads like you may be a third party auditor. Perhaps that's intentional. 

Should iDP (OIDC provider) compromise or insider attack be a factor? They could mint id tokens for any account they wish to without needing the account owner to be part of the flow?

Reply:
> I may be pedantic here, so don't spin on it, but might be worth defining who 'we' is, as it reads like you may be a third party auditor. Perhaps that's intentional.

[Bad habit from academic writing](https://www.quora.com/Is-it-okay-to-use-we-in-scientific-writing) 😄 

> Should iDP (OIDC provider) compromise or insider attack be a factor? They could mint id tokens for any account they wish to without needing the account owner to be part of the flow?

I think we cover that below in the table, but it's kinda hard to read. We'll see if we can make that clearer.